api/services/Flairs.js

```diff
@@ -73,26 +76,26 @@ exports.applied = function (apps, flair) {
 };
 exports.inPokemonTradesTrader = function (flair) {
   if (flair) {
-    return flair.sub === "pokemontrades" && !flair.involvement && !flair.giveaways;
+    return flair.sub === configLocal.reddit.tradeSub && !flair.involvement && !flair.giveaways;
   }
 };
 exports.inPokemonTradesHelper = function (flair) {
   if (flair) {
-    return flair.sub === "pokemontrades" && (flair.involvement > 0 || flair.giveaways > 0);
+    return flair.sub === configLocal.reddit.tradeSub && (flair.involvement > 0 || flair.giveaways > 0);
   }
 };
 exports.inSVExchangeHatcher = function (flair) {
   if (flair) {
-    return flair.sub === "svexchange" && flair.eggs > 0;
+    return flair.sub === configLocal.reddit.eggSub && flair.eggs > 0;
   }
 };
 exports.inSVExchangeGiver = function (flair) {
   if (flair) {
-    return flair.sub === "svexchange" && flair.giveaways > 0;
+    return flair.sub === configLocal.reddit.eggSub && flair.giveaways > 0;
   }
 };
 exports.userHasFlair = function (user, flair) {
-  if (flair.sub === 'pokemontrades') {
+  if (flair.sub === configLocal.reddit.tradeSub) {
```

Comment:
Breaks Can apply for involvement flair with all the prerequisites + Cannot downgrade flair